### PR TITLE
update companies add lendis.io

### PR DESCRIPTION
**Pull request by null9beta**
Adds the following events or companies:
 - Lendis

### Checklist

 - [x] I have updated the event or company counts
 - [x] I have put the most recent relevant date in the "Last Update" column
 - [ ] I have linked to the article about the change, not the company's website

The article has not yet being finished, will update as soon as available

### CODE REVIEW

**Comment by drewbrew** [comment]
The change looks fine, but it has a merge conflict that GitHub won't let me resolve. 

Would you mind taking care of it?

Thanks!

**Comment by null9beta** [comment]
@drewbrew surely. here you go

File: _data/companies.yml [hunk 1]
```diff
@@ -1503,8 +1503,8 @@
   events: Discouraged
   last_update: 2020-03-12
 
-- name: Urjanet
-  wfh: Optional
+- name: Lendis
+  wfh: Required
```
Comment (drewbrew) [inline _data/companies.yml]: This is deleting Urjanet from the list of companies, which we don't want to do.

Comment (null9beta) [reply _data/companies.yml]: oh, sry my bad, fixing it

Comment (null9beta) [reply _data/companies.yml]: @drewbrew fixed

**Pull request merged**